### PR TITLE
Newcomers_Guide.rst: Remove bolding from heading

### PR DESCRIPTION
**Pull request by sidharthskumar**
i removed the double quotes that enclosed :- pull request and also
deleted extra unwanted underline below :- pull request

Fixes https://github.com/coala/coala/issues/4442

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by ihsavru** [comment]
You have already created a pull request for this issue. Please close the last one. When you create commits they are automatically added to your PR so you don't have to create a new PR everytime :)

**Comment by JohnMoutafis** [comment]
ack 3f91800

**Comment by sils** [comment]
ack 3f91800

The commit message lacks a verb FWIW but this'd be rather nitpicky

**Comment by sils** [comment]
ack 3f91800 ?

**Comment by sils** [comment]
@cherry-pick-bot you there?

**Comment by sidharthskumar** [comment]
Should I edit my commit message? If yes , please tell me .

**Comment by sils** [comment]
@sidharthskumar I think something like `Newcomers_Guide.rst: Remove bolding from heading` or so would be better because it explains what you're doing.

**Comment by sidharthskumar** [comment]
Sure I ll do it right away

On 06-Jul-2017 4:55 pm, "Lasse Schuirmann" <notifications@github.com> wrote:

@sidharthskumar <https://github.com/sidharthskumar> I think something
like Newcomers_Guide.rst:
Remove bolding from heading or so would be better because it explains what
you're doing.

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/coala/coala/pull/4447#issuecomment-313369444>, or mute
the thread

**Comment by sidharthskumar** [comment]
@sils you mean just the heading right? or about changing full message body?

**Comment by sils** [comment]
just the heading (called shortlog)

Sincerely,

Lasse Schuirmann

lasse@schuirmann.net
http://coala.io/ - http://viperdev.io/ - http://gitmate.com/

**Comment by sidharthskumar** [comment]
Okay thank you

Sincerely,
Sidharth

**Comment by sidharthskumar** [comment]
can anyone please tell me why this is not getting merged :( , should it be reacke'd?

**Comment by sils** [comment]
Hm, this has conflicts :/ it needs a rebase with a conflict resolution, we can only merge PRs that are fully up to date (which this wasn't), sorry for communicating unclear.

**Comment by sidharthskumar** [comment]
Oh, so Should I just rebase it through git ?

**Comment by sils** [comment]
Yep

**Comment by sidharthskumar** [comment]
now what ? :( feeling so bad :(

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -313,8 +313,13 @@ where ``myfork`` is the name of your fork that you added at the previous step.
     You can also add a profile picture to your Github account so that
     you can stand out from the crowd!
 
+<<<<<<< HEAD
```
Comment (JohnMoutafis) [inline docs/Developers/Newcomers_Guide.rst]: You need to remove those

File: docs/Developers/Newcomers_Guide.rst [hunk 2]
```diff
@@ -313,8 +313,13 @@ where ``myfork`` is the name of your fork that you added at the previous step.
...
 Step 5. Creating a ``Pull Request``
 -----------------------------------
+=======
```
Comment (JohnMoutafis) [inline docs/Developers/Newcomers_Guide.rst]: Same as above

File: docs/Developers/Newcomers_Guide.rst [hunk 3]
```diff
@@ -313,8 +313,13 @@ where ``myfork`` is the name of your fork that you added at the previous step.
...
+Step 6. Creating a Pull Request
+-------------------------------
+>>>>>>> Newcomers_Guide.rst: Step 6 out of place bold text
```
Comment (JohnMoutafis) [inline docs/Developers/Newcomers_Guide.rst]: Same as the previous issues

Comment (sidharthskumar) [inline docs/Developers/Newcomers_Guide.rst]: can i close this PR and open a new one? its getting quite messy giving more errors as i push it.

Comment (sidharthskumar) [inline docs/Developers/Newcomers_Guide.rst]: hey john i was wondering if this errors are caused naturally or was it my mistake somewhere? i am actually a newcomer; if this happens almost all  the time (at least for such a simple issue) then wouldnt it be cubersome while fixing other difficulty/high issues ? just curious :(

**Comment by sils** [comment]
hey, you have a git conflict because someone edited some stuff in the same
document that you touched and that was merged in the meantime, just google
git conflicts :)

**Comment by sidharthskumar** [comment]
Oh ok . Thank you John 😅

**Comment by sidharthskumar** [comment]
@sils oh i see.. thank you very much , hope this time it gets merged :D ;) .

**Comment by jayvdb** [comment]
@sidharthskumar, you have two problems:

1. two commits in https://github.com/coala/coala/pull/4447/commits ; there should only be one commit.
2. there is a merge conflict in your commits.  They are the `<<<<<<< HEAD` you can see in https://github.com/coala/coala/pull/4447/commits/5a7d326a6d99453d7adc2c6e7f86c903fb73ca8d

**Comment by jayvdb** [comment]
unack 5a7d326 0f82e24

**Comment by sidharthskumar** [comment]
thank you for bearing with me. Now i have come to know about many stuffs and will look forward not to repeat these mistakes again . i have tried to delete the commit whole night but  failed to do so. So i closed this PR and opened a new one here https://github.com/coala/coala/pull/4456 .

**Pull request closed without merging**